.github/workflows/callable-create-github-container-image-frontend.yml

```
@@ -0,0 +1,79 @@
+
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+  build:
+    permissions:
+      packages: write
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: './${{ inputs.package-dir }}'
```

actionlint (1.7.4)

property "package-dir" is not defined in object type {service: string; tag: string}

(expression)